### PR TITLE
Added a title message on join. Closes #679.

### LINKED ISSUE #679: Title on Join
On join, a message could pop up saying "Hello," etc.

### PR DESCRIPTION
**Pull request by Hockeyfan360**
This commit adds a basic title message whenever a player joins. It also adds a method to the utilities class. This has been tested, and it works.

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java [hunk 1]
```diff
@@ -910,6 +910,8 @@ public void run()
                 }
             }
         }.runTaskLater(TotalFreedomMod.plugin, 20L * 1L);
+        TFM_Util.sendTitle(player, "§6§lWelcome back,", 2, 2, 2);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: Please use `ChatColor` for colors.

File: src/me/StevenLawson/TotalFreedomMod/TFM_Util.java [hunk 2]
```diff
@@ -1003,6 +1009,27 @@ public static void reportAction(Player reporter, Player reported, String report)
             }
         }
     }
+    
+    //Note that in order for the titles to work properly, the sub-title must follow the title packet.
+    public static void sendTitle(Player player, String message, int fadein, int stay, int fadeout)
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/TFM_Util.java]: Methods that require nms classes should be in the TFM_ServerInterface class. :)

File: src/me/StevenLawson/TotalFreedomMod/TFM_Util.java [hunk 3]
```diff
@@ -1003,6 +1009,27 @@ public static void reportAction(Player reporter, Player reported, String report)
...
+    {
+        CraftPlayer craftplayer = (CraftPlayer) player;
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/TFM_Util.java]: In case this fails casting (running on some other platform), fail gracefully. :)

**Comment by mysteriace** [comment]
This is a good idea, it just needs to be redone.

**Comment by JeromSar** [comment]
@mysteriace This is a PR, not an issue. It is being worked on (see code above).

**Comment by Hockeyfan360** [comment]
I am going to close this, and open a new PR.

**Pull request closed without merging**